src/a-d/Alligator/Alligator.StreamHub.cs

```diff
@@ -3,7 +3,7 @@ namespace Skender.Stock.Indicators;
 // WILLIAMS ALLIGATOR (STREAM HUB)
 
 /// <summary>
-/// Streaming hub for calculating Alligator indicator.
+/// Streaming hub for Alligator indicator.
 /// </summary>
 /// <inheritdoc cref="IAlligator"/>
```

Comment:
This inheritance of the interface class is done inconsistently.
I also don't think it technically does anything really, right?
Decide if this is a good pattern or not.  I do not currently see any benefit.
If it's valuable and adding end-user value then do it consistently on all.